docs/changes.rst

```diff
@@ -785,7 +785,7 @@ Released on August 11th 2017.
 * The max-length checks is now enforcing length of text in form.
 * Make the commit_pending age configurable per component.
 * Various user interface cleanups.
-* Fixed component/project/sitewide search for translations.
+* Fixed component/project/site wide search for translations.
```

Comment:
“site-wide” is a compound adjective and should be hyphenated.

Reply:
Do you know if it should be "site-wide" when it isn't transitive? Some places it is given as a description between "local" "string only" and ~"for the entire platform/instance".
From what I can tell it is "command line", but then "command-line client"

Reply:
Sometimes, the adjective may be omitted from the phrase but still be implied (“site-wide [setting]”), so I would still hyphenate it.

Reply:
This is definitely used in more places, so let's deal with "site-wide" spelling separately, I will merge this PR to get other fixes in.